[2025-04-21T23:37:40.817+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T23:37:40.853+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews_pipeline.model_data manual__2025-04-21T23:36:55.641358+00:00 [queued]>
[2025-04-21T23:37:40.873+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews_pipeline.model_data manual__2025-04-21T23:36:55.641358+00:00 [queued]>
[2025-04-21T23:37:40.876+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-21T23:37:40.911+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): model_data> on 2025-04-21 23:36:55.641358+00:00
[2025-04-21T23:37:40.926+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=492) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-21T23:37:40.929+0000] {standard_task_runner.py:72} INFO - Started process 507 to run task
[2025-04-21T23:37:40.931+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews_pipeline', 'model_data', 'manual__2025-04-21T23:36:55.641358+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpoa_o7lmd']
[2025-04-21T23:37:40.933+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask model_data
[2025-04-21T23:37:40.992+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews_pipeline.model_data manual__2025-04-21T23:36:55.641358+00:00 [running]> on host 95fe2406fe61
[2025-04-21T23:37:41.141+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews_pipeline' AIRFLOW_CTX_TASK_ID='model_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-21T23:36:55.641358+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-21T23:36:55.641358+00:00'
[2025-04-21T23:37:41.144+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-21T23:37:41.146+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-21T23:37:41.147+0000] {logging_mixin.py:190} INFO - Current task name:model_data state:running start_date:2025-04-21 23:37:40.854878+00:00
[2025-04-21T23:37:41.147+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews_pipeline and current dag run status:running
[2025-04-21T23:37:41.148+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-21T23:37:41.163+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-04-21T23:37:41.184+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-21T23:37:41.201+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-21T23:37:41.203+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews_pipeline, task_id=model_data, run_id=manual__2025-04-21T23:36:55.641358+00:00, execution_date=20250421T233655, start_date=20250421T233740, end_date=20250421T233741
[2025-04-21T23:37:41.241+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-21T23:37:41.243+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-21T23:37:41.243+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews_pipeline queued_at:2025-04-21 23:36:55.658570+00:00
[2025-04-21T23:37:41.244+0000] {logging_mixin.py:190} INFO - Task hostname:95fe2406fe61 operator:PythonOperator
[2025-04-21T23:37:41.269+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-21T23:37:41.295+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-21T23:37:41.298+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
